liquity/dev
# Enable bLUSD liquidity management with LUSD zaps

# How to test this PR locally

In liquity/dev

Run a fork of mainnet via hardhat:
```
ALCHEMY_API_KEY=your_api_key yarn start-fork
```

Run the frontend in dev mode:
```
yarn start-demo:dev-frontend
```

In liquity/ChickenBond

Deploy BLUSDLPZap.sol to your mainnet fork:
```
forge create --rpc-url http://localhost:8545 --private-key 0x4d5db4107d237df6a3d58ee5f70ae63d73d7658d4026f2eefd2f204c81682cb7 src/BLUSDLPZap.sol:BLUSDLPZap
```

In your web browser

Get your random wallet address from the browser console on http://localhost:3000:
```
account
```

In liquity/dev

Give yourself some bLUSD
```
cd packages/lib-ethers
npx hardhat setStorageSlotIndex --contract-address 0xB9D7DdDca9a4AC480991865EfEf82E01273F79C3 --wallet-address YOUR_WALLET_ADDRESS_FROM_BROWSER_CONSOLE --network forkedMainnet
```

Then take out a Trove on Liquity to obtain some LUSD and now you should be able to test the Liquiduity Mangement feature of LUSD bonds.
